zrml/swaps/src/tests.rs

```diff
@@ -1033,6 +1033,51 @@ fn swap_exact_amount_out_exchanges_correct_values_with_rikiddo() {
     });
 }
 
+#[test]
+fn create_pool_fails_on_too_many_assets() {
+    ExtBuilder::default().build().execute_with(|| {
+        let length = 12;
+        let assets: Vec<Asset<MarketId>> =
+            (0..length).map(|x| Asset::CategoricalOutcome(0, x)).collect::<Vec<_>>();
```

Comment:
We should use the crate configuration value that's specified within `mock.rs` for that
```suggestion
        let length = <Runtime as crate::Config>::MaxAssets::get();
        let assets: Vec<Asset<MarketId>> =
            (0..=length).map(|x| Asset::CategoricalOutcome(0, x)).collect::<Vec<_>>();
```